app/views/game_search/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<header class='masthead'>
+<header class='quiz'>
```

Comment:
I want game association pages to look more uniform so I changed it to match the trees with our game show pages